src/main/java/Duke.java

```diff
@@ -1,10 +1,56 @@
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String userInput;
+        int taskIndex=0;
+        Scanner in = new Scanner(System.in);
+        Task[] listOfTasks=new Task[100];
+        System.out.println("Hello! I'm Groot");
+        System.out.println("What can I do for you?");
+        userInput = in.nextLine();
+        while(!userInput.equalsIgnoreCase("bye")){
+            if(!userInput.equalsIgnoreCase("list")){
+                if(userInput.length()>4 && userInput.contains("done")){
+                    int doneTaskNo=Integer.parseInt(userInput.substring(5))-1;
```

Comment:
Would it be better to read if the variables and System.out were separated with a blank line?